4.5 release page (#1138)
godotengine/godot-website: collections/_release_4_5/entry-systems-navigation-process-navigation-regions-asynchronously.md

---
type: entry
section: systems
subsection: navigation
rank: 1
importance: 3
anchor: process-navigation-regions-asynchronously
title: Process navigation regions asynchronously
text: |

>   The main thread of a software is like a project leader. If the project leader handles too much tasks and doesn't delegate enough, it can affect the overall performance of the team.

Reviewer: ```suggestion
  The main thread of a software is like a project leader. If the project leader handles too many tasks and doesn’t delegate enough, it can affect the overall performance of the team.
```

>   The main thread of a software is like a project leader. If the project leader handles too much tasks and doesn't delegate enough, it can affect the overall performance of the team.

Reviewer 2: ```suggestion
  The main thread of a software is like a project leader. If the project leader handles too many tasks and doesn't delegate enough, it can affect the overall performance of the team.
```

>   The main thread of a software is like a project leader. If the project leader handles too many tasks and doesn’t delegate enough, it can affect the overall performance of the team.

Reviewer 2: ```suggestion
  The main thread of a process is like a project leader. If the project leader handles too many tasks and doesn’t delegate enough, it can affect the overall performance of the team.
```
I didn't like "a software". I'm not sure if process is correct, though.

Reviewer: I'd say "a program" or "an application", a thread can run as a process and the line is pretty thin between the two I'd say

Reviewer 2: Yep, either of those would be good.

Author: I used "a computer program", just to be sure.

>   Enabling async iterations asks the navigation servers to delegate the navigation process to a background thread.

Reviewer 3: ```suggestion
  Enabling async iterations asks the navigation servers to delegate the navigation process to a background thread, which can improve overall navigation performance.
```
With this added, the sentence below could be removed

>   Using this new feature can overall improve navigation performance.

Reviewer 2: ```suggestion
  Using this new feature can improve overall navigation performance.
```

Author: As I used the word "overall" in the first paragraph, I replaced that sentence with:
> Using this new feature can improve navigation performance in most cases.
